sql/src/test/java/io/crate/integrationtests/SysNodeResiliencyIntegrationTest.java

```diff
@@ -77,4 +76,16 @@ public void testTimingOutNode() throws Exception {
             waitNoPendingTasksOnAll();
         }
     }
+
+    /**
+     * Test cluster name check, when path.data does not contain cluster
+     * name as a folder.
+     *
+     * TODO-ES6: Remove this when ES >= 6.0
+     */
+    @Test
+    public void testClusterNameInPathCheckDoesNotFailWithDefaultConfig() throws Exception {
```

Comment:
btw this test is actually a bit missleading - it tests that the default test-setup contains a non-deprecated data path. But doesn't really verify that our regular default config is okay.